### PR TITLE
docs(behavior_velocity_planner): use markdown extensions for image alignment

### PR DESCRIPTION
**Pull request by satoshi-ota**
Signed-off-by: satoshi-ota <satoshi.ota928@gmail.com>

## Description

images are not shown in https://autowarefoundation.github.io/autoware.universe/main/planning/behavior_velocity_planner/crosswalk-design/

I fixed following points:

- not use `<div>` tag
- use mkdocs-material https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by satoshi-ota** [comment]
FYI:https://mkdocs.readthedocs.io/en/859/user-guide/configuration/#use_directory_urls

File: mkdocs.yaml [hunk 1]
```diff
@@ -63,6 +63,7 @@ markdown_extensions:
       guess_lang: false
   - fontawesome_markdown
   - footnotes
+  - md_in_html
```
Comment (kenji-miyake) [inline mkdocs.yaml:66]: Could you add a reference link for this extension?

Comment (kenji-miyake) [reply mkdocs.yaml:66]: Also, please write some description of this PR.

**Comment by yukkysaito** [comment]
@satoshi-ota I cannot see the figure. is it correct?
image

**Comment by satoshi-ota** [comment]
> @satoshi-ota I cannot see the figure. is it correct? image

@yukkysaito yes, we can't see figures correctly in github.com such as plantuml, but these figs are shown in external documentation site.
https://autowarefoundation.github.io/autoware.universe/pr-1299/planning/behavior_velocity_planner/crosswalk-design/

**Comment by satoshi-ota** [comment]
Pass required CI checks :smile: 
Could you give me any comment or Approve ?

**Comment by satoshi-ota** [comment]
> Pass required CI checks smile Could you give me any comment or Approve ?

@yukkysaito @kenji-miyake @TakaHoribe

**Review by TakaHoribe — APPROVED** [review]
LGTM

File: mkdocs.yaml (hunk 1 above)
Comment (kenji-miyake) [reply mkdocs.yaml:66]: https://squidfunk.github.io/mkdocs-material/reference/images/#configuration

**Pull request merged**